tests/e2e/networks.json

```diff
@@ -0,0 +1,19 @@
+[
```

Comment:
this file should probably not be checked in or people might f*** with our tests by draining the accounts.

Reply:
ok, we can leave the wallets in for now. I think it is cleaner though if we remove the whole file and add it via env variable later